src/stylesheets/components/_navigation.scss

```
@@ -1,41 +1,59 @@
 .app-navigation {
+  border-bottom: 1px solid $govuk-border-colour;
   background-color: $app-light-grey;
 
+  @include govuk-media-query($from: tablet) {
+    display: block;
+  }
+}
+
+.no-js .app-navigation {
   @include govuk-media-query($until: tablet) {
-    display: none;
+    display: block;
   }
 }
 
 .app-navigation__list {
-  position: relative;
-  left: -(govuk-spacing(3));
   margin: 0;
   padding: 0;
   list-style: none;
+
+  @include govuk-media-query($from: tablet) {
+    position: relative;
+    left: govuk-spacing(-3);
+  }
 }
 
 .app-navigation__list-item {
-  $navigation-height: 50px;
+  position: relative;
 
-  @include govuk-font(19, $weight: bold, $line-height: $navigation-height);
+  @include govuk-media-query($from: tablet) {
+    $navigation-height: 50px;
 
-  box-sizing: border-box;
-  display: block;
-  position: relative;
-  height: $navigation-height;
-  height: govuk-px-to-rem($navigation-height); // sass-lint:disable-line no-duplicate-properties
-  padding: 0 govuk-spacing(3);
-  float: left;
+    @include govuk-font(19, $weight: bold, $line-height: $navigation-height);
+    box-sizing: border-box;
+    height: $navigation-height;
+    padding: 0 govuk-spacing(3);
+    height: govuk-px-to-rem($navigation-height); // sass-lint:disable-line no-duplicate-properties
```

Can we keep the `height` rules together?

```suggestion
    height: $navigation-height;
    height: govuk-px-to-rem($navigation-height); // sass-lint:disable-line no-duplicate-properties
    padding: 0 govuk-spacing(3);
```